dev/scripts/src/alluxio.org/build/cmd/build.go

```diff
@@ -127,6 +127,7 @@ func buildTarball(opts *buildOpts) error {
 		cmd := command.New(baseMvnCmd).WithDir(repoBuildDir)
 		if !opts.suppressMavenOutput {
 			cmd = cmd.SetStdout(os.Stdout)
+			cmd = cmd.SetStderr(os.Stderr)
```

Comment:
@Xenorith this helped me troubleshooting my mvn issues (cuz i'm new to building dora until this time). Actually I feel stderr is more helpful than stdout